src/cpp_output.cpp

```diff
@@ -1126,7 +1126,8 @@ void cpp_output::gen_action_part_script(scxml_parser::action &a)
 	const string expr = a.attr["expr"];
 
 	//out << tab << tab << "// " << a.type << " expr=" << expr << endl;
-	out << tab << tab << expr << endl;
+	//out << tab << tab << expr << endl;
+	out << regex_replace(expr, regex("(^|\n)"), string("$1") + tab + tab) << endl;
```

Comment:
This modifies the code more than you might want.
```c++
std::cout << "Hello \
world!\n";
```
becomes 
```c++
		std::cout << "Hello \
		world!\n";
```
which modifies the string.

The user's code should not be touched, even if the indenting is not perfect.

Reply:
Thanks. You are right - I will change that back

Reply:
Of cause, lines ending with '\' could be skipped for indenting. But maybe there are other unexpected cases.

Reply:
Checking for `\` should work. I have no idea, where it would fail.